source/blog/2018-05-30-emberjs-accessibility-and-a11y-tools.md

```diff
@@ -0,0 +1,153 @@
+
+While ember-a11y refers to a whole bunch of tools, there's one addon that is simply called [ember-a11y](https://github.com/ember-a11y/ember-a11y) too. The main task of this addon is to handle focus correctly so that users with Assistive Technology can navigate content as changes happen inside the `{{outlet}}` of a route. 
+
+```
+ember install ember-a11y
+```
+Install it and replace instances of `{{outlet}}` with 
+`{{focusing-outlet}}`.
+
+So why is this necessary? If you're looking at a decently accessible website and you hit the tab key, you'll see the focus jump between different elements - things get highlighted. Front end frameworks like Ember sometimes get in the way of the DOM structure that works best for screen reader focus, since as you move between routes, only part of the page changes. The contents of `{{outlet}}` are dynamic. 
+
+The problem is that screen readers rely heavily on what has focus. Imagine if you had to start reading at the very top of the page whenever a route in your app changed, instead of just reading the content that is new, or if you weren't able to tell that some content had changed! ember-a11y corrects for that focus problem so that Assistive Technology navigation using focus works correctly. From the README:
+
+> The current implementation of this addon will immediately apply focus to the most relevant piece of content based on actions users take (clicking buttons, links, etc). This allows screen readers to catch changes and read the right information, thus providing a much better experience for blind users.
+
+## What next?
+
+It's our responsibility as developers to build inclusive tech, so keep learning! [Melanie Sumner's](https://github.com/MelSumner) EmberConf talk [Accessibility for All](https://www.youtube.com/watch?v=6ydNf7e5P00) is a fast, concise introduction to accessibility in Ember, and [this article series](http://www.melsumner.com/blog/accessibility/a11y-accessibility-guide-ember-developers/) provides a deeper dive. There are also more ember-a11y addons available that solve different issues. 
+
+Almost all of us will have a disability at some point in our lives. Accessible web practices lead to better user experience - things like improved keyboard navigation, text that is still readable on a dim monitor, and clear pointers on how to interact with a UI. 
+
+Here are some ways you can help improve accessibility in Ember and the JavaScript ecosystem:
+- Research and try out other [ember-a11y](https://github.com/ember-a11y) tools
+- Make your own apps accessible
+- Include a11y considerations in your talks and articles
+- Contribute to addons that help solve these problems
+- Guide your coworkers and collaborators through the learning process
+- Avoid language that perpetuates the idea that accessibility is too hard. Focus
+on the tools that make it easier and our responsibilities as developers.
+- Lead an RFC to make Ember more accessible out of the box
```

Comment:
this one is already in progress, maybe we can remove this line

Reply:
Changed to "Support and comment on RFCs that help make Ember more accessible out of the box"